### PR TITLE
[Migration] Add deleted column to pod versions

### PR DESCRIPTION
**Pull request by segiddins**
\c @floere

### CODE REVIEW

**Comment by orta** [comment]
This needs a change to lib/migrate.rb too bumping the version number, see https://github.com/CocoaPods/Humus/commit/c14e888e9565a4b496a57e02fa8beb02e7716a8c for similar

File: migrations/schema.txt [hunk 1]
```diff
@@ -117,6 +117,7 @@ pod_versions
 | created_at | 1114 | "timestamp without time zone" | nil                                        | true       | false       | :datetime | nil          |
 | updated_at | 1114 | "timestamp without time zone" | nil                                        | true       | false       | :datetime | nil          |
 | pod_id     | 23   | "integer"                     | nil                                        | false      | false       | :integer  | nil          |
+| deleted    | 16   | "boolean"                     | "false"                                    | true       | false       | :boolean  | false        |
```
Comment (kylef) [inline migrations/schema.txt]: Shouldn't this have an `is_` prefix like the other boolean fields?

- [`is_active`](https://github.com/CocoaPods/Humus/blob/master/migrations/schema.txt#L199)
- [`is_verified`](https://github.com/CocoaPods/Humus/blob/master/migrations/schema.txt#L100)

Comment (kylef) [inline migrations/schema.txt]: heh, second though. We have other boolean uses without `is_`. This seems a mess.

**Comment by orta** [comment]
Yeah, lets do this.

**Comment by floere** [comment]
@kyle Yeah, `is_` seems like unnecessary noise.

**Comment by floere** [comment]
Is this NOT NULL? (Unsure what type boolean translates to)

**Comment by kylef** [comment]
@floere In that case, should we be changing the other booleans by removing any `is_` prefixes?

**Comment by floere** [comment]
@kylef Personally, I would, but with low prio.

**Comment by floere** [comment]
@segiddins I'll add specs tomorrow my morning - that should help with the code :)

**Comment by floere** [comment]
@segiddins I've set the column to be non-null. Big plus from me and go for migration :) :shipit:

**Comment by segiddins** [comment]
@floere doesn't this close https://github.com/CocoaPods/Humus/issues/14 ?

**Comment by floere** [comment]
@segiddins Only when it is live.

**Comment by orta** [comment]
Alright, so, what do we want to do here?

**Comment by orta** [comment]
I read the PR, I don't think anything needs to happen. Merging and will deploy.

**Comment by floere** [comment]
Still gets a go from me! :ship:

**Comment by orta** [comment]
Verified locally, and have just deployed + migrated. This is in.

**Comment by floere** [comment]
Just noticed my derpy commit message 😅 Oh well…

**Comment by floere** [comment]
Thanks!

**Pull request merged**